guides/common/modules/proc_synchronizing-a-content-view-to-a-smartproxy.adoc

```diff
@@ -0,0 +1,40 @@
+[id="Synchronizing_a_Content_View_to_a_{smart-proxy-context}{context}"]
+= Synchronizing a content view to a {SmartProxy}
+
+You can use the Hammer CLI to synchronize a content view to a {SmartProxy}.
```

Comment:
```suggestion
You can use the Hammer CLI to synchronize a content view to a {SmartProxy}.
```

Doesn't this happen automatically? Why do users have to do that? And under what circumstances?

Reply:
Users can only sync all selected lifecycle environments at the same time via the UI. So, in hammer, we let them sync smaller things: single lifecycle environments, single content views, and single repositories.
You'd want to do it if you only want to update a small specific set of content on the smart proxy.

Reply:
@bangelic The information from Ian looks useful for the user. Please, incorporate it into the introductory paragraph. Let's explain why using Hammer for the syncing is more beneficial than using the web UI, ie. it provides more fine-grained control over the sync.